src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -107,11 +107,18 @@ function ChartDeploymentHistory({
             })
     }, [])
 
+    const getDeploymentData = (_selectedDeploymentTabIndex: number, _selectedDeploymentHistoryIndex: number) => {
+        if (_selectedDeploymentTabIndex === 1 || _selectedDeploymentTabIndex === 2) {
```

Comment:
why this special condition?

Reply:
To check for source & hem manifest. Added supportive constant